catboost/tutorials/ranking/ranking_tutorial.ipynb

```diff
@@ -615,7 +615,7 @@
     "        for doc_id, line in enumerate(f):\n",
     "            line = line.split(',')[:2]\n",
     "            \n",
-    "            label, query_id = tuple(map(float, line))\n",
+    "            label, query_id = float(line[0]), int(line[1])\n",
```

Comment:
The line was changed because the current version of Pool doesn't accept float group_id and raises `CatBoostError: group_id[0] object (1.0) is unsuitable (should be string or integral type)`